radio/src/translations/ua.h

```diff
@@ -0,0 +1,1257 @@
+#define TR_MENUMODELSEL                TR("ВИБІРМОД", "ВИБІР МОДЕЛІ")
+#define TR_MENU_MODEL_SETUP            TR("НАЛАШТУВ", "НАЛАШТУВАННЯ МОДЕЛІ")
+#if defined(SURFACE_RADIO)
+#define TR_MENUFLIGHTMODES             "РЕЖИМИ ВОДІННЯ"
+#define TR_MENUFLIGHTMODE              "РЕЖИМ ВОДІННЯ"
+#else
+#define TR_MENUFLIGHTMODES             "РЕЖИМИ ПОЛЬОТУ"
+#define TR_MENUFLIGHTMODE              "РЕЖИМ ПОЛЬОТУ"
+#endif
+#define TR_MENUHELISETUP               "РЕЖИМ ГЕЛІКОПТЕРА"
+
+  #define TR_MENUINPUTS                "ВХОДИ"
+  #define TR_MENULIMITS                "ВИХОДИ"
+#define TR_MENUCURVES                  "КРИВІ"
+#define TR_MENUCURVE                   "КРИВА"
+#define TR_MENULOGICALSWITCH           "ЛОГІЧНІ ПЕРЕМИКАЧ"
+#define TR_MENULOGICALSWITCHES         "ЛОГІЧНІ ПЕРЕМИКАЧІ"
+#define TR_MENUCUSTOMFUNC              "СПЕЦІАЛЬНІ ФУНКЦІЇ"
+#define TR_MENUCUSTOMSCRIPTS           "КОРИСТУВАЦЬКИ СКРИПТИ"
+#define TR_MENUTELEMETRY               "ТЕЛЕНМЕТРІЯ"
+#define TR_MENUSTAT                    "СТАТИСТИКА"
+#define TR_MENUDEBUG                   "ДЕБАГ"
+#define TR_MONITOR_CHANNELS1           "МОНИТОР КАН 1-8"
+#define TR_MONITOR_CHANNELS2           "МОНИТОР КАН 9-16"
+#define TR_MONITOR_CHANNELS3           "МОНИТОР КАН 17-24"
+#define TR_MONITOR_CHANNELS4           "МОНИТОР КАН 25-32"
+#define TR_MONITOR_SWITCHES            "МОНИТОР ЛОГІЧНИХ ПЕРЕМИКАЧІВ"
+#define TR_MONITOR_OUTPUT_DESC         "Виходи"
+#define TR_MONITOR_MIXER_DESC          "Мікшери"
+  #define TR_RECEIVER_NUM              TR("RxNum", "No. приймача")
+  #define TR_RECEIVER                  "Приймач"
+#define TR_MULTI_RFTUNE                TR("Налаштув.частоти", "Точне налаштування Радіочастоти")
+#define TR_MULTI_RFPOWER               "RF power"		/* use english */
+#define TR_MULTI_WBUS                  "Вихід"
+#define TR_MULTI_TELEMETRY             "Телеметрія"
+#define TR_MULTI_VIDFREQ               TR("Від.Част.", "Відео частота")
+#define TR_RF_POWER                    "RF Power"	/* use english */
+#define TR_MULTI_FIXEDID               TR("FixedID", "Fixed ID")		/* use english */
+#define TR_MULTI_OPTION                TR("Опція", "Значення опції")
+#define TR_MULTI_AUTOBIND              TR(INDENT "Прив'язка на кан.", INDENT "Прив'язка на канал")
+#define TR_DISABLE_CH_MAP              TR("Нема.кан.карти", "Вимкнути карту каналів")
+#define TR_DISABLE_TELEM               TR("Нема телеметрії", "Вимкнути телеметрію")
+#define TR_MULTI_DSM_AUTODTECT         TR(INDENT "Автовизначення", INDENT "Формат автовизначення")
+#define TR_MULTI_LOWPOWER              TR(INDENT "Низька потужн.", INDENT "Ркжим нізької потужності")
+#define TR_MULTI_LNA_DISABLE           INDENT "Вимкн. LNA"
+#define TR_MODULE_TELEMETRY            TR(INDENT "S.Port", INDENT "S.Port link")		/* use english */
+#define TR_MODULE_TELEM_ON             TR("ON", "Увімкнено")
+#define TR_DISABLE_INTERNAL            TR("Дект. внутр.", "Деактивувати внутр. RF")
```

Comment:
There is a typo in a shortened version, I would suggest to replace it with a different word standing for `Turning off`:
```suggestion
#define TR_DISABLE_INTERNAL            TR("Викл. внутр.", "Виключити внутрішній RF")
```